### ISSUE TITLE
Vue appereance editable via config-value

### ISSUE THREAD

**Issue by Khepri-Design** [tobimori/kirby-icon-field#10]
Maybe it is a good idea to somehow store in the config if you want to color the fill or the stroke from the icons in the panel.
For e.g. if you use feather icons, the iconfont has outlines only, so the icons currently appear as "white blocks" in the dropdown.

**Comment by tobimori** [comment]
The plugin does not change the fill color, it only uses `filter` to change it to monochrome white/black. This has to be an issue with the panel styles itself.

Can you check that?

**Comment by Khepri-Design** [comment]
correct, maybe i had something runnning in the background. Couldnt reproduce it at the monent. Can be closed i guess
